js-miniapp-sample/src/js_sdk/index.js

```diff
@@ -0,0 +1,5 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const getUUIDFromMobileSdk = () => {
+  return uuidv4();
```

Comment:
We've merged the JS SDK portion now, so could you please integrate that instead?

Because we're using Yarn workspaces, you should be able to use just `import MiniApp from "js-miniapp-sdk"`. Also, the `getUniqueId` function uses promises, so you will need to make some modifications to handle this.

Reply:
@corycaywood 
will getUniqueId respond with uuid? i see no implementation on bridge?

Reply:
@Rajesh-Sundaram The implementation is on the Android/iOS side, so you won't be able to test in the browser. We have been considering making some kind of mocking functionality for browser testing, but this hasn't been implemented yet.

For now, probably you will need to first check if it's running on a mobile device (perhaps using 'bowser'? https://lancedikson.github.io/bowser/docs/Parser.html#getPlatform) and return a mock value if it's not on mobile.

Reply:
added